### ISSUE TITLE
Add ability to NOT defend

### ISSUE THREAD

**Issue by coltonw** [coltonw/sensors-are-down#36]
Players should be able to choose not to defend when they do not want to.

**Comment by mturley** [comment]
What do you get to do instead of defending? Or is it just a "pass", where you lose that opportunity and gain nothing?

**Comment by coltonw** [comment]
Correct.  What you "get" is that you didn't use any card up, since cards are the only limited resource in the game, it may be worth it to hold onto 2 strategically significant cards and not defend, especially in cases where perhaps you already have a good defense set up.
